semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Index.scala

```diff
@@ -1,40 +0,0 @@
-package scala.meta.internal.metacp
-
-import org.langmeta.internal.io._
-import org.langmeta.io._
-import scala.collection.mutable
-import scala.meta.internal.{semanticdb3 => s}
-
-class Index {
```

Comment:
What is supposed to replace this class? I was trying to rebase this PR on top of master but it fails compilation because this class is missing.

Reply:
Oops. Looks like the file got accidentally gitignored on my machine. Fixed & pushed.